data/json/items/ammo.json

```diff
@@ -285,9 +285,11 @@
         "ammo_type" : "BB",
         "damage" : 2,
         "range" : 12,
-        "dispersion" : 11,
-        "count" : 200,
-        "effects" : ["NEVER_MISFIRES"]
+        "dispersion" : 15,
```

Comment:
Hold on. At some point most most of the dispersion was moved from ammunition to the guns themselves. 15 is still too low for a sphere traveling through space out of an unrifled barrel. The paper cartridge might be a good model for bb's base dispersion.